packages/components/navbar/stories/Navbar.stories.tsx

```diff
@@ -37,7 +37,7 @@ const Account = ({
 }: Partial<NavbarAccountProps>) => (
   <Navbar.Account
     username="username"
-    avatar="https://images.ctfassets.net/iq4lnigp6fgt/2EEEk92Kiz6KxREsjBLPAN/810d5a21650d91abad12e95da4cd3beb/2021-06_Everyone_is_Welcome_here_1_.png?fit=fill&f=top_left&w=48&h=48"
+    avatar="https://www.gravatar.com/avatar/f6d4b637888d9c661921216832a9126e?s=50&d=https%3A%2F%2Favatars.contentful.com%2Finitials%2Fv1%2FJD%2F50"
```

Comment:
```suggestion
    avatar="https://avatars.contentful.com/initials/v1/JD/50"
```

I would suggest we don't use third-party assets since they go down once in a while and it leads to Chromatic snapshots breaking. This is not really a big concern for gravatar, but still good practice.

Reply:
Thank's i was not aware of that URL, just copied the one from my avatar in the app 😅

Reply:
TIL too!